docs/ams-schema-spec.bs

```diff
@@ -1,174 +1,1208 @@
 <pre class='metadata'>
-Title: Amsterdam Schema Specification
+Title: Amsterdam Schema Specificatie
 Shortname: ams-schema-spec
-Revision: 1.1.1
+Revision: 2.0.0
 Status: LS
 URL: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html
 Repository: Amsterdam/amsterdam-schema
 Editor: Michiel Trimpe, Gemeente Amsterdam, https://data.amsterdam.nl/
-Abstract: Het Amsterdam Schema is een standaard voor JSON Schema's welke gebruikt kan worden om
-    datasets in detail op machine readable formaat te kunnen omschrijven zodat deze generiek
-    verwerkt en ontsloten kunnen worden.
+Abstract: Het Amsterdam Schema is een op JSON Schema’s gebaseerde standaard (ondersteund door het Amsterdam Meta Schema)
+    waarmee datasets in een machine readable formaat tot in detail beschreven kunnen worden.
+    Zo kunnen deze datasets op generieke wijze verwerkt en ontsloten worden.
 Markup Shorthands: css no, markdown yes
 Indent: 4
-Default highlight: json
+Default highlight: js
+Logo: https://api.data.amsterdam.nl/api/gemeente_Amsterdam_logo.svg
+Favicon: https://data.amsterdam.nl/favicon.png
 Boilerplate: copyright no, footer no
+Max ToC Depth: 2
+Text Macro: DCAT <a data-link-type=biblio data-lt="vocab-dcat-2" data-biblio-type=informative>DCAT</a>
+Text Macro: DCATURL https://www.w3.org/TR/vocab-dcat-2/
+Text Macro: BP <blockquote class="best-practice"> **Best Practice**
+Text Macro: BPE </blockquote>
+Text Macro: CONST *constant*
+Text Macro: HIDELINESEXAMPLE class="example hide-lines" onClick="this.classList.toggle('hide-lines');this.classList.toggle('unhide-lines');"
+Text Macro: SYNCSCROLLA class="a" onscroll="let sibling =this.nextElementSibling; sibling.scrollTop=this.scrollTop;sibling.scrollLeft=this.scrollLeft;"
+Text Macro: SYNCSCROLLB class="b" onscroll="let sibling =this.previousElementSibling; sibling.scrollTop=this.scrollTop;sibling.scrollLeft=this.scrollLeft;"
 </pre>
 
-
+<!--
+████ ██    ██ ████████ ████████   ███████
+ ██  ███   ██    ██    ██     ██ ██     ██
+ ██  ████  ██    ██    ██     ██ ██     ██
+ ██  ██ ██ ██    ██    ████████  ██     ██
+ ██  ██  ████    ██    ██   ██   ██     ██
+ ██  ██   ███    ██    ██    ██  ██     ██
+████ ██    ██    ██    ██     ██  ███████
+-->
 Introductie {#intro}
 ====================
+> **Doelstelling** Het doel van Amsterdam Schema is:
+    Alle binnen de Gemeente Amsterdam beschikbare data binnen één generiek universeel ondersteund, machine readable formaat beschrijven.
```

Comment:
Het doel van het Amsterdam Schema is om:

- Alle data binnen gemeente Amsterdam te ontsluiten via een machine verwerkbaar universeel formaat
- Metadata specificaties in te zetten als instructie voor geautomatiseerde dataverwerking en ontsluiting
- Bij te dragen aan een datagedreven organisatie

-

Reply:
Het ontstuiten vind alleen niet plaats binnen amsterdam schema zelf, maar het overkoepelend doel ontbreekt nu idd. Wat vind je hier van?

Het doel van Amsterdam Schema is om:  
Alle data van gemeente Amsterdam binnen één machine verwerkbaar universeel formaat beschrijven, zodat het geautomatiseerd verwerkt en ontsloten kan worden. En zo vanuit de [tada principes](https://www.amsterdam.nl/innovatie/digitalisering-technologie/tada-principes/)  bij te dragen aan een transparante, datagedreven organisatie.